### DISCUSSION TITLE
Add the ability to schedule unpublish in a workflow

### DISCUSSION THREAD

**Discussion by sniffdk** [umbraco/Umbraco.Workflow.Issues#43] [Ideas]
When using the Workflow package, it's possible to set a schedule date when an editor requests to publish a page. However, at the moment, it is not possible to also set a scheduled unpublish date.
It would be beneficial to have this option, so the suggestion is simply to add that functionality to the Workflow package.

**Comment by nathanwoulfe** [comment]
Hey @sniffdk, to be honest, I can't remember why that wasn't an option in the first place... May have been a reason, maybe not. It does make sense, will see what I can make work.

**Reply by sniffdk** [reply]
Thanks @nathanwoulfe , that sounds great, hopefully it isn't too much work 😅

**Reply by nathanwoulfe** [reply]
Pretty sure it's all good to go - did require a bit of refactoring to split the single scheduledDate property into a releaseDate and expireDate, but that's good change as it aligns better with the CMS. Plan is for an RC next week, keep an eye out for the blog post / comms.

**Reply by sniffdk** [reply]
That is fantastic news @nathanwoulfe, thank you! Looking forward to the RC 🥳 🙏
